packages/cli/src/cache.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+import fs from 'fs';
+import path from 'path';
+import crypto from 'crypto';
```

Comment:
TODO: check if `crypto` is faster than `murmur`hash which is what we use for hashing classNames etc.

We should use whatever is fastest because we don't care about the cryptographic properties of hashing algorithms that are hard to reverse-engineer.